src/olympia/addons/tests/test_cron.py

```diff
@@ -151,6 +151,34 @@ def test_move_disabled_file(self, mv_mock):
                                    self.msg)
         assert mv_mock.call_count == 1
 
+    @mock.patch('olympia.files.models.storage.exists')
+    @mock.patch('olympia.files.models.move_stored_file')
+    def test_move_disabled_addon_ioerror(self, mv_mock, storage_exists):
+        # raise an IOError for the first file, we need to make sure
+        # that the second one is still being properly processed
+        mv_mock.side_effect = [IOError, None]
+        storage_exists.return_value = True
+
+        # Use Addon.objects.update so the signal handler isn't called.
+        Addon.objects.filter(id=self.addon.id).update(
+            status=amo.STATUS_PUBLIC, disabled_by_user=True)
+        File.objects.update(status=amo.STATUS_PUBLIC)
+
+        cron.hide_disabled_files()
+
+        # Check that f2 was moved.
+        f2 = self.f2
+        mv_mock.assert_called_with(f2.file_path, f2.guarded_file_path)
+
+        # Check that f1 was moved as well.
```

Comment:
Could you update that comment that the file wasn't actually moved, but that we called `move_file` on it?